src/stdlib/parse_auditd.rs

```diff
@@ -0,0 +1,432 @@
+use std::collections::BTreeMap;
+
+use crate::{btreemap, compiler::prelude::*};
+use bytes::BytesMut;
+use chrono::DateTime;
+use linux_audit_parser::{Number, Value as AuditValue};
+
+fn parse_auditd(bytes: Value) -> Resolved {
+    let bytes = bytes.try_bytes()?;
+    // check if bytes ends with newline, otherwise append it
+    // TODO: make the parser accept bytes without newline in the linux_audit_parser crate (to-be-contributed)
```

Comment:
Agreed, it seems like the parser should work without trailing newlines.

Reply:
https://github.com/hillu/linux-audit-parser-rs/pull/3